### ISSUE TITLE
Ajax Request

### ISSUE THREAD

**Issue by faguiargptools** [laraflock/dashboard#9]
I have issues with ajax requests, the RoleMiddleware.php have a code line:

``` PHP
if ($request->ajax()) {
    return response('Unauthorized', 401);
}
```

 How can I solve this?

**Comment by iolson** [comment]
I will be updating this in a future release when I create the PermissionMiddleware. If you update to v3.0.3, I moved the middleware to being used in the actual routes instead of the controller, so you could "for now" copy over RouteMiddleware to your own custom middlware and remove the ajax request.

You can see how you can use your own custom routes, or move the routes over to the applications `app/Http/routes.php` file, and disable the package routes via the config file.

I will update this though, most likely by this weekend.

**Comment by iolson** [comment]
Fixed in v3.1.0
